cmd/cdi/clients/github.go

```diff
@@ -0,0 +1,70 @@
+package clients
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
+)
+
+type Client interface {
```

Comment:
This interface is what we can build out. Thinking a little forward.. might not be even needed if we just smoosh everything into 1 GraphQL Query. 🤷‍♂️